pytorch_blade/torch_blade_build.py

```diff
@@ -59,6 +59,18 @@ def __init__(
         self.static_tensorrt = check_env_flag(
             "TORCH_BLADE_BUILD_TENSORRT_STATIC", "OFF"
         )
+        self.blade_gemm = check_env_flag(
+            "TORCH_BLADE_BUILD_BLADE_GEMM", "OFF"
+        )
+        self.blade_gemm_nvcc = os.getenv(
+            "TORCH_BLADE_BUILD_BLADE_GEMM_NVCC", "/usr/local/cuda-11.6/bin/nvcc"
```

Comment:
Why should this `nvcc` be specific to 11.6? I prefer to move those less general configurations from `bazel_setup.py` script to `.bazelrc`. I mean these environment variables:
`TORCH_BLADE_BUILD_BLADE_GEMM_NVCC_ARCHS`
`TORCH_BLADE_BUILD_BLADE_GEMM`
`TORCH_BLADE_BUILD_BLADE_GEMM_NVCC`
`TORCH_BLADE_BUILD_BLADE_GEMM_LIBRARY_KERNELS`

Reply:
As we have disscuss offline, make these action_envs into internal .bazelrc.

Reply:
As we have disscuss offline, make these action_envs into internal .bazelrc.